subsys/mgmt/ec_host_cmd/ec_host_cmd_handler.c

```diff
@@ -244,20 +246,31 @@ int ec_host_cmd_init(struct ec_host_cmd_backend *backend)
 
 	handler_tx_buf = hc->tx.buf;
 	handler_rx_buf = hc->rx_ctx.buf;
+	handler_tx_buf_end = handler_tx_buf + CONFIG_EC_HOST_CMD_HANDLER_TX_BUFFER_SIZE;
+	handler_rx_buf_end = handler_rx_buf + CONFIG_EC_HOST_CMD_HANDLER_RX_BUFFER_SIZE;
 
 	ret = backend->api->init(backend, &hc->rx_ctx, &hc->tx);
 
+	backend_tx_buf = hc->tx.buf;
+	backend_rx_buf = hc->rx_ctx.buf;
+
 	if (ret != 0) {
 		return ret;
 	}
 
-	if (!hc->tx.buf | !hc->rx_ctx.buf) {
+	if (!hc->tx.buf || !hc->rx_ctx.buf) {
```

Comment:
Use your helper variable consistently.
```suggestion
	if (!backend_tx_buf|| !backend_rx_buf) {
```

Reply:
Done.